File: agahkeyboard/src/main/java/nfkeyboard/models/networkmodels/Photo.java
package nfkeyboard.models.networkmodels;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import nfkeyboard.models.AllSuggestionModel;

/**
 * Created by Shimona on 02-06-2018.
 */

public class Photo {
    @SerializedName("ImageURL")
    @Expose
    private String imageUri;

    @SerializedName("selected")
    @Expose
    private boolean selected;

    public String getImageUri() {
        return imageUri;
    }

    public void setImageUri(String imageUri) {
        this.imageUri = imageUri;
    }

    public boolean getSelected() {
        return selected;
    }

    public void setSelected(boolean selected) {
        this.selected = selected;
    }

    public AllSuggestionModel toAllSuggestion() {
        AllSuggestionModel model = new AllSuggestionModel(imageUri);
        model.setImageUri(getImageUri());
        model.setSelected(getSelected());
        return model;
    }

}

File: florisboard/src/main/java/dev/patrickgold/florisboard/customization/model/response/Photo.kt
package dev.patrickgold.florisboard.customization.model.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import dev.patrickgold.florisboard.R
import dev.patrickgold.florisboard.customization.adapter.BaseRecyclerItem
import dev.patrickgold.florisboard.customization.adapter.FeaturesEnum

data class Photo(
  @SerializedName("ImageURL")
  @Expose
  var imageUri: String? = null,
  @SerializedName("selected")
  @Expose
  var selected: Boolean = false,

  var gridType: ViewGridType = ViewGridType.FOUR_GRID
) : BaseRecyclerItem() {
  override fun getViewType(): Int = FeaturesEnum.PHOTOS.ordinal

  enum class ViewGridType(var width:Int,var countGrid:Int,var icon:Int){
    FIRST_GRID(280,1, R.drawable.ic_layout_one_f),SECOND_GRID(210,2,R.drawable.ic_layout_two_f),THIRD_GRID(164,3,R.drawable.ic_layout_three_f),FOUR_GRID(132,4,R.drawable.ic_layout_four_f),
  }
}